cdap-api/src/main/java/co/cask/cdap/api/workflow/AbstractWorkflowAction.java

```diff
@@ -49,7 +50,7 @@ public void destroy() {
     // No-op
   }
 
-  protected final WorkflowContext getContext() {
+  public WorkflowContext getContext() {
```

Comment:
The WorkflowDriver doesn't pass in the context object, it's actually created only in the initialize method.  If I don't have it public, what's another way to get the context back? (so I can do this in getTransactionContextFromAction)